doc/dev/mgmt/tests.md

```diff
@@ -69,18 +69,26 @@ export AZURE_TENANT_ID=<value>      # Linux shell only
 ```
 *** Note: when setting these variables, do not wrap the value in quotation marks ***
 
-You are now able to log in from Python using OAuth.
+You are now able to log in from Python using OAuth. Before use, you need to run the `pip install azure-identity` command to install it.
 You can test with this code:
 ```python
 import os
-from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import ClientSecretCredential
 
-credentials = ServicePrincipalCredentials(
+credentials = ClientSecretCredential(
     client_id = os.environ['AZURE_CLIENT_ID'],
     secret = os.environ['AZURE_CLIENT_SECRET'],
     tenant = os.environ['AZURE_TENANT_ID']
 )
 ```
+Or you can use `DefaultAzureCredential`, which we prefer. 
```

Comment:
we should notice users that they also need to set env environment before using `DefaultAzureCredential`